edx_exams/apps/api/serializers.py

```diff
@@ -1,4 +1,33 @@
-# Serializers that can be shared across multiple versions of the API
-# should be created here. As the API evolves, serializers may become more
-# specific to a particular version of the API. In this case, the serializers
-# in question should be moved to versioned sub-package.
+"""
+Serializers for the edx-exams API
+"""
+from rest_framework import serializers
+
+from edx_exams.apps.core.models import Exam
+
+
+class ExamSerializer(serializers.ModelSerializer):
+    """
+    Serializer for the Exam Model
+    """
+
+    exam_name = serializers.CharField(required=True)
+    course_id = serializers.CharField(required=True)
```

Comment:
There's no validation that all of the course_id's within a request have to match. Should we validate that?

Reply:
That's not something you can do in this service right? Leave that to the LMS service instead.

Reply:
the URL already gives us the course ID, there isn't actually a need for it in the serialized object

Reply:
I like that, I'll remove it here. I wanted to keep it in the URL in case someone sends an empty request (which should then mark all exams as inactive).